driver-core/src/test/resources/client-side-encryption/aggregate.json

```diff
@@ -150,18 +150,6 @@
             "command_name": "listCollections"
           }
         },
-        {
-          "command_started_event": {
-            "command": {
-              "listCollections": 1,
-              "filter": {
-                "name": "datakeys"
-              },
-              "$db": "keyvault"
-            },
-            "command_name": "listCollections"
-          }
-        },
```

Comment:
Just confirming my understanding. Is this correct that this expectations were removed because they are gathered from a user-facing `MongoClient`, which does not use the `keyvault` DB anymore (the DB is now used by an internal `MongoClient` owned by a `Crypt` object)?

Reply:
Exactly.